programs/helium-entity-manager/src/constants.rs

```diff
@@ -1 +1 @@
-pub static HOTSPOT_METADATA_URL: &str = "https://entities.nft.test-helium.com";
+pub static HOTSPOT_METADATA_URL: &str = "https://entities.nft.helium.io";
```

Comment:
Should this be a conditional based on TESTING env var?

Reply:
Good idea, made it off of devnet feature flag